src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -48,10 +48,11 @@ public boolean maintainRPM() {
     if(shooterMaster.getSelectedSensorVelocity() < setPoint){
       shooterMaster.set(ControlMode.PercentOutput, 1);
     } else {
-      shooterMaster.set(ControlMode.PercentOutput, .85);
+      shooterMaster.set(ControlMode.PercentOutput, .65);
```

Comment:
Sam, we picked 0.65 at random, you should tune to get this value: new battery, set shooter voltage so that we get <70k open loop